sandbox/src/Links.js

```diff
@@ -14,13 +14,38 @@ export default function Links() {
     <div>
       <h2>Links</h2>
       <p>
-        This page tests collecting events on link clicking. For example, this
-        link:{" "}
+        This page shows a few different ways link clicks can be handled in
+        Alloy.
+      </p>
+      <p>
+        When clicking on this <a href="https://example.com">exit link</a>, Alloy
+        records and transmits an exit-link web interaction event.
+      </p>
+      <p>
+        When clicking on this{" "}
+        <a href="download.zip" target="_blank">
+          download link
+        </a>
+        , Alloy records and transmits a download-link web interaction event.
+      </p>
+      <p>
```

Comment:
Can you add an example where the download property is set on a link?

Reply:
I will add this to the documentation instead.